clients/src/main/java/org/apache/kafka/common/metrics/Metrics.java

```diff
@@ -332,7 +332,7 @@ public synchronized Sensor sensor(String name, MetricConfig config, long inactiv
                     children.add(s);
                 }
             }
-            log.debug("Added sensor with name {}", name);
+            log.trace("Added sensor with name {}", name);
```

Comment:
It was too noisy at debug level when i was trying to debug. But i didn't mean to commit it